src/org/opendatakit/briefcase/ui/automation/AutomationPanelForm.java

```diff
@@ -0,0 +1,164 @@
+package org.opendatakit.briefcase.ui.automation;
+
+import static org.opendatakit.briefcase.ui.reused.FileChooser.directory;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.function.Consumer;
+import javax.swing.JButton;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+import org.opendatakit.briefcase.automation.AutomationConfiguration;
+import org.opendatakit.briefcase.export.ExportConfiguration;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
+import org.opendatakit.briefcase.ui.export.components.ConfigurationDialog;
+import org.opendatakit.briefcase.util.StringUtils;
+
+@SuppressWarnings("checkstyle:MethodName")
+public class AutomationPanelForm {
+  public JPanel container;
+  private JLabel scriptDirLabel;
+  private JTextField scriptDirField;
+  private JButton scriptDirChooseButton;
+  private JButton generateScriptButton;
+  private JButton changeDefaultConfigurationButton;
+
+  private Optional<ExportConfiguration> exportConfiguration = Optional.empty();
+
+  public AutomationPanelForm() {
+    $$$setupUI$$$();
+    scriptDirChooseButton.addActionListener(__ ->
+        directory(container, fileFrom(scriptDirField))
+            .choose()
+            .ifPresent(file -> setScriptDir(Paths.get(file.toURI()))));
+    changeDefaultConfigurationButton.addActionListener(__ -> setExportConfiguration());
+    if (exportConfiguration.isPresent()) {
+      changeDefaultConfigurationButton.setText("Change export configuration");
+    }
+  }
+
+  private void setExportConfiguration() {
+    ConfigurationDialog dialog = ConfigurationDialog.from(exportConfiguration, true, BriefcasePreferences.getStorePasswordsConsentProperty());
+    dialog.onOK(config -> {
```

Comment:
You can contract this into `dialog.onOK(config -> exportConfiguration = Optional.ofNullable(config));`